plasma_core/transaction_v2.py

```diff
@@ -0,0 +1,88 @@
+import rlp
+from rlp.sedes import big_endian_int, binary, CountableList
+from ethereum import utils
+from plasma_core.utils.signatures import sign, get_signer
+from plasma_core.utils.transactions import encode_utxo_id
+from plasma_core.constants import NULL_SIGNATURE, NULL_ADDRESS
+
+
+def pad_list(to_pad, value, required_length):
+    return to_pad + [value] * (required_length - len(to_pad))
+
+
+class TransactionInput(rlp.Serializable):
+
+    fields = (
+        ('blknum', big_endian_int),
+        ('txindex', big_endian_int),
+        ('oindex', big_endian_int)
+    )
+
+    def __init__(self, blknum=0, txindex=0, oindex=0):
+        self.blknum = blknum
+        self.txindex = txindex
+        self.oindex = oindex
+
+    @property
+    def identifier(self):
+        return encode_utxo_id(self.blknum, self.txindex, self.oindex)
+
+
+class TransactionOutput(rlp.Serializable):
```

Comment:
Just a side note - pyRLP has changed its API in 1.0.0, breaking things. Should we merge this one first or move to pyRLP 1.0.0 first? What do you prefer?